src/server/handlers/github.js

```diff
@@ -64,13 +70,19 @@ export const listRepositories = (req, res) => {
         });
       }
 
-      return res.status(response.statusCode).send({
+      let body = {
         status: 'success',
         payload: {
           code: 'github-list-repositories',
           repos: response.body
         }
-      });
+      };
+
+      if (response.headers.link) {
+        body.pageLinks = parseLinkHeader(response.headers.link);
+      }
```

Comment:
This could do with corresponding changes to the route documentation in `docs/routes.md`.